pages/admin/directory.tsx

```diff
@@ -813,10 +765,12 @@ const MemberEdit: FC<{
             <div className="relative flex flex-col gap-2">
               <Input
                 name="email"
+                className={
+                  email && email.email !== originalEmail && "text-brown-600"
```

Comment:
Nice catch, this color change matching the rest feels good.

Noticing we could probably take another pass at how we indicate that fields modified, but don't want to block this PR. Created an issue to track https://github.com/hawaiiansintech/hawaiiansintech/issues/34